cmd/misc/main.go

```diff
@@ -221,6 +221,32 @@ func updateAggreationBits(rpcClient *rpc.LighthouseClient, startEpoch uint64, en
 					})
 					continue
 				}
+
+				status := uint64(0)
+				err := db.ReaderDb.Get(&status, `
+				SELECT status
+				FROM blocks WHERE 
+					slot=$1`, block.Slot)
+				if err != nil {
+					utils.LogError(err, fmt.Errorf("error getting Slot [%v] status", block.Slot), 0)
+					return
+				}
+
+				if status != block.Status {
+					logrus.Infof("Slot[%v] has the wrong status [%v], but should be [%v]", block.Slot, status, block.Status)
+					if block.Status == 1 {
+						err := db.SaveBlock(block)
+						if err != nil {
+							utils.LogError(err, fmt.Errorf("error saving Slot [%v]", block.Slot), 0)
+							return
+						}
+						continue
```

Comment:
**question:** Why do we continue here to the next block after updating the slot?

Reply:
Because all the relevant information is set in the db.SaveBlock(block), so we don't have to check the aggregation bits explicitly for this one.

Reply:
Are you sure this works?
db.SaveBlock(block) does save the aggregation bits here
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/11ccd3cd738899a84b800a5009ec7d6849df3a19/db/db.go#L1515
but if the entry already exists we have an `ON CONFLICT ... DO NOTHING`

Reply:
But we only get here for blocks with status 0 in our db - which means we have not yet saved the block data for this block.   So there should not be any conflicts.

Reply:
I see. Thank you for the explanation.